material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex

```diff
@@ -1,116 +1,240 @@
-\begin{itemize}
-\item Introduce \boldDarkRed{unknown} measures for $Q$'s position from $N_o$ to help you express \posvec{N_o}{Q}.
-\item Write \posvec{B_o}{Q} as an \boldDarkRed{unknown} multiple of $\uvecWithHat{u}^{Q/B_o}$.
-\item Write a \boldDarkRed{vector loop} equation.
-\end{itemize}
-\end{minipage}
-\hfill
-\begin{minipage}[t]{0.35\linewidth}
-\flushright
-\vspace*{0pt}
-\includegraphics[width=\linewidth]{football.png}
+  \item
+  %\\[0.5pc]
+  \begin{minipage}[t]{0.6\linewidth}
+    When the football is lying on the field, the operator clicks on the
+    football in the video. Assume the click is converted into a direction
+    vector pointing to $Q$ from $B_o$ as
+    $\uvecWithHat{u}^{Q \division B_o} \equals 1~\uvecx{b}$.
+    \\[0.5pc]
+    Find a \textbf{system of \boldUnderlineDarkRed{scalar} equations} that can be
+    used to solve for the \underline{\uvecx{n} and \uvecy{n} measures} of the
+    football's position from $N_o$. The rotation table will be formed later; for
+    now you may leave unknown dot-products un-evaluated
+    (such as $\uvecx{b} \cdot \uvecx{n}$).
+    \\[0.5pc]
+    To do this, it may help to:
+      \begin{itemize}
+        \item Introduce \boldDarkRed{unknown} measures for $Q$'s position
+        from $N_o$ to help you express \posvec{N_o}{Q}.
+        \item Write \posvec{B_o}{Q} as an \boldDarkRed{unknown} multiple of
+        $\uvecWithHat{u}^{Q \division B_o}$.
+        \item Write a \boldDarkRed{vector loop} equation.
+      \end{itemize}
+  \end{minipage}
+  \hfill
+\begin{minipage}[t]{0.44\linewidth}
+  \minipageTopAnchor
+  %\vspace*{-1cm}
+  \begin{center}
+    \includegraphicsAB[width=0.66\linewidth]
+      {football1.png}
+      {football.png}
+  \end{center}
 \end{minipage}
-
+\Solution{}{0.99\linewidth}{
+  \begin{minipage}{0.65\linewidth}
+     \begin{tabular}{@{ }l@{}r@{ \equals[\;] }ll}
+       &\posvec{N_o}{Q}& $x\uvecx{n} \plus y\uvecy{n} \plus o\uvecz{n}$
+       \\[1.0pc]
+       &\posvec{N_o}{B_o}& $50\uvecx{n} \plus 0\uvecy{n} \plus 20\uvecz{n}$
+       \\[1.0pc]
+       &\posvec{B_o}{Q} & $\lambda\uvecx{b}$
+       \\[1.0pc]
+       Loop: & \posvec{N_o}{Q}&\posvec{N_o}{B_o}\plus\posvec{B_o}{Q}
+       \\[1.0pc]
+       &$x\uvecx{n} \plus y\uvecy{n}$& $50\uvecx{n} \plus
+       20\uvecz{n} \plus\lambda\uvecx{b}$
+       \\[1.0pc]
+       \end{tabular}
+       \begin{tabular}{c@{}l}
+       \begin{tabular}{lr@{ \equals[\;] }l}
+       dot w/ \uvecx{n}: & x & $ 50 \plus \lambda \uvecx{b} \DotProduct \uvecx{n} $
+       \\[1.0pc]
+       dot w/ \uvecy{n}: & y & $ 0 \plus \lambda \uvecx{b} \DotProduct \uvecy{n} $
+       \\[1.0pc]
+       dot w/ \uvecz{n}: & o & $ 20 \plus \lambda \uvecx{b} \DotProduct \uvecz{n} $
+       \\[1.0pc]
+       \end{tabular}
+       &
+       $\begin{array}{l}
+       \MyLBrace{8.9ex}{scalar system w/ 3 equations, 3 unknowns x,y,$\lambda$}
+       \end{array}$
+       \end{tabular}
+       \end{minipage}
+       \\[0.45pc]
+       }
```

Comment:
The `\end{}` macros and the closing `}` should match the indentation level of the corresponding `\begin{}` and `\Solution` above.